keras_cv/layers/preprocessing/color_jitter.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class ColorJitter(layers.Layer):
+    """ColorJitter class randomly apply brightness, contrast, saturation 
+    and hue image processing operation sequentially and randomly on the input. 
+    It expects input as RGB image. The expected image should be `(0-255)` pixel ranges.
+
+    Input shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
+    Output shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
+
+    Args:
+        brightness_factor: A non-negative single float value. It will be used 
```

Comment:
I think it is better to make all those 4 param in the same fashion:
1. Accept single int or pair of int
2. Value range from either [0, 1] or [-1, 1]. When [0, 1] is the default range, then single int input is treat as hight bound. Otherwise the single input x indicate the value to be [-x, x].

Reply:
Yeah, lets always conform to [0, 1] with 0 representing the weakest transform and 1 the strongest.  I have updated the API design guidelines to include this point.  https://github.com/keras-team/keras-cv/blob/master/.github/API_DESIGN.md#strength-parameters